lib/manager/gradle-lite/readme.md

```diff
@@ -0,0 +1,7 @@
+This is an alternate manager for Gradle, written completely in JavaScript.
+
+It performs much faster as it doesn't execute Gradle command.
+
+You can enable both `gradle` and `gradle-lite` if you wish.
+
+If commits produced by `gradle-lite` look good to you, you are welcome to switch completely.
```

Comment:
```suggestion
`gradle-lite` is an an alternate manager for Gradle, and is written in JavaScript.
The main benefit of `gradle-lite` is that it skips the slow Gradle commands.

You can use the default `gradle` manager and `gradle-lite` at the same time.

If you like the commits from `gradle-lite`, you can use `gradle-lite` as a complete replacement for the default manager.
```

How about something like this?
I don't know anything about the code, so correct anything that I'm getting wrong.

Reply:
Sounds good, gradle is also a memory killer causing code spaces to OOM 🙃